sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateFormatterSuite.scala

```diff
@@ -208,6 +208,6 @@ class DateFormatterSuite extends DatetimeFormatterSuite {
     val errMsg = intercept[DateTimeException] {
       formatter.parse("x123")
     }.getMessage
-    assert(errMsg.contains("Cannot cast x123 to DateType"))
+    assert(errMsg.contains("Invalid `date` literal: 'x123'"))
```

Comment:
Please, use `toSQLType` to output types, see https://github.com/apache/spark/pull/36233

Reply:
Will do, thanks for the source!